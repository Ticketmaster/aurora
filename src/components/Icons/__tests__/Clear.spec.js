/* eslint-disable import/no-extraneous-dependencies */
import React from "react";
import renderer from "react-test-renderer";

import { ClearIcon } from "../";

describe("ClearIcon", () => {
  const children = <title>Clear Icon</title>;

  it("renders correctly", () => {
    const component = renderer.create(<ClearIcon>{children}</ClearIcon>);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
