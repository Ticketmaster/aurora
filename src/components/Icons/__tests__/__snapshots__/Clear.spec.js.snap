// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClearIcon renders correctly 1`] = `
<svg
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    Clear Icon
  </title>
  <g
    fill="none"
    fillRule="evenodd"
  >
    <path
      d="M0 0h24v24H0z"
    />
    <path
      d="M12 22c5.523 0 10-4.477 10-10S17.523 2 12 2 2 6.477 2 12s4.477 10 10 10zm0 1C5.925 23 1 18.075 1 12S5.925 1 12 1s11 4.925 11 11-4.925 11-11 11zm3.89-15.596a.5.5 0 0 1 .706.707l-8.485 8.485a.5.5 0 0 1-.707-.707l8.485-8.485zm-8.486.707a.5.5 0 0 1 .707-.707l8.485 8.485a.5.5 0 0 1-.707.707L7.404 8.111z"
      fill="currentColor"
      fillOpacity=".5"
      fillRule="nonzero"
    />
  </g>
</svg>
`;
